db/re/item_combo_db.txt

```diff
@@ -1073,7 +1073,6 @@
 20135:20136,{ bonus bAllStats,12; }
 2053:19451,{ bonus2 bSubRace,RC_Player_Human,3; bonus2 bAddClass,Class_All,15; bonus2 bMagicAddClass,Class_All,15; }
 2053:27301,{ bonus2 bAddClass,Class_All,10; bonus bAspdRate,10; }
-2060:15388:15389,{ .@sum = getequiprefinerycnt(EQI_HAND_R)+getequiprefinerycnt(EQI_ARMOR)+getequiprefinerycnt(EQI_GARMENT); bonus bMatk,30; autobonus "{ bonus2 bMagicAtkEle,Ele_Earth,10; bonus2 bMagicAtkEle,Ele_Wind,10; }",10,10000,BF_MAGIC; if (.@sum>=30) { bonus2 bMagicAddEle,Ele_Undead,20; bonus2 bMagicAddEle,Ele_Holy,20; } }
```

Comment:
Autobonus rate of the remaining entry looks wrong.
It has autobonus of 0,1% chance, but DP description says 10%.

Reply:
You misread, the rates are not in the description, they are set to the minimum until we get them
```
When equipped with King Schmidt's Suit and King Schmidt's Manteau, Matk + 30, has a chance to gain bonus wind and earth property magical damage by 10% for 10 seconds when dealing magical damage.
```

Reply:
Ah yeah my bad. Sorry!